code/unclassified/lapindrom_checker/lapindrome_checker.cpp

```diff
@@ -0,0 +1,35 @@
+// part of cosmos by opengenus foundation
```

Comment:
In my opinion, you could store to code/utility/lapindrome_checker/lapindrome_check.cpp.

Reply:
@iattempt its fine here coz its an unclassified algo not utility :smile: